docs/enhancements/on-prem-data-collection.md

```diff
@@ -0,0 +1,220 @@
+---
+title: on-prem-data-collection
+authors:
+  - "@CrystalChun"
+creation-date: 2022-09-30
+last-updated: 2022-10-28
+---
+
+# On-premises Data Collection
+
+## Summary
+
+The Assisted Installer ("AI") will provide a process to collect data about installations when it is deployed on-premises as part of either [Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6) ("RHACM") or as a stand-alone Podman deployment. Data was already collected from clusters deployed on [Red Hat's Hybrid Cloud Console](https://console.redhat.com/openshift/assisted-installer/clusters) (“SaaS”), but there was no method for obtaining data from on-premises deployments.
```

Comment:
The epic mentions BILLI, are we considering that here as well?

Reply:
Good question, @celebdor @filanov is BILLI part of the scope of this epic?

Reply:
Yes